recipes/libgettext/all/conanfile.py

```diff
@@ -92,7 +92,7 @@ def build(self):
             elif self.settings.arch == "x86_64":
                 host = "x86_64-w64-mingw32"
                 rc = "windres --target=pe-x86-64"
-            automake_perldir = tools.unix_path(os.path.join(self.deps_cpp_info['automake'].rootpath, "bin", "share", "automake-1.16"))
+            automake_perldir = tools.unix_path(os.path.join(self.deps_cpp_info['automake'].rootpath, "res", "automake-1.16"))
```

Comment:
I think you can use the `AUTOMAKE_PERLLIBDIR` environment variable.
See https://github.com/conan-io/conan-center-index/blob/c25a202acf6bf13c9ca33b324f8e1f15bb21ea0d/recipes/automake/all/conanfile.py#L127-L129

Reply:
Or even better, use `self.deps_user_info_build.compile`:
https://github.com/conan-io/conan-center-index/blob/c25a202acf6bf13c9ca33b324f8e1f15bb21ea0d/recipes/automake/all/conanfile.py#L137